pages/hackathon.tsx

```diff
@@ -87,33 +92,49 @@ export default function HackathonPage({ pageTitle }) {
             Moʻokūʻauhau &amp; Moʻōlelo
           </DataListItem>
         </DataList>
-        {/* <DataList mainEventLogistics gap="1rem 1rem">
-          <Button
-            size={ButtonSize.Small}
-            customWidth="18rem"
-            customFontSize="1.5rem"
-          >
-            RSVP
-          </Button>
+        <DataList mainEventLogistics gap="1rem 1rem">
+          <a href={EVENTBRITE_LINK}>
+            <Button
+              size={ButtonSize.Small}
+              customWidth="16rem"
+              customWidthSmall="28rem"
+              customFontSize="1.5rem"
+            >
+              Buy Tickets
+            </Button>
+          </a>
           <a href={DISCORD_URL}>
             <Button
-              customWidth="18rem"
+              customWidth="16rem"
+              customWidthSmall="20rem"
               customFontSize="1.5rem"
               size={ButtonSize.Small}
               variant={ButtonVariant.Secondary}
             >
               Join the Discord
             </Button>
           </a>
-          <Button
-            customWidth="5rem"
-            customFontSize="1.5rem"
-            size={ButtonSize.Small}
-            variant={ButtonVariant.Secondary}
-          >
-            ...
-          </Button>
-        </DataList> */}
+          <Dropdown
+            links={{
+              "purplemaia.org": "https://purplemaia.org/",
+              "@purple_maia": "https://twitter.com/purple_maia/",
+              purple_maia: "https://www.instagram.com/purple_maia/",
+              "@hawaiiansintech": "https://twitter.com/hawaiiansintech/",
+              hawaiiansintech: "https://www.instagram.com/hawaiiansintech/",
```

Comment:
Is it possible to add the Twitter and Instagram icons here to disambiguate the two? https://fontawesomeicons.com/instagram